src/libraries/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -139,6 +197,11 @@ private bool WaitOneNoCheck(int millisecondsTimeout)
             {
                 if (success)
                     waitHandle.DangerousRelease();
+
+                if (isWaitHandleKeywordEnabled)
+                {
+                    NativeRuntimeEventSource.Log.WaitHandleWaitStop();
+                }
```

Comment:
This can be deleted, it's left over from before and is replaced by the changes in my commit

Reply:
Oops sorry about, I'll fix all that now.